src/styles/index.scss

```diff
@@ -1,3 +1,47 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  padding: 10px;
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  width: (100px + 10px) * 7;
```

Comment:
The width calculation `(100px + 10px) * 7` is intended to set the calendar's width based on the day block size and gap. However, the gap is defined separately using `gap: 1px;`. Consider adjusting the calculation to account for the actual gap size or ensure consistency between the width and gap settings.